build/pgo/Terminal.PGO.props

```diff
@@ -0,0 +1,45 @@
+<Project>
+  <PropertyGroup>
+    <NuGetPackageDirectory>$(MSBuildThisFileDirectory)..\..\packages</NuGetPackageDirectory>
+    <PkgMicrosoft_PGO_Helpers_Cpp>$(NuGetPackageDirectory)\Microsoft.PGO-Helpers.Cpp.0.2.22</PkgMicrosoft_PGO_Helpers_Cpp>
+  </PropertyGroup>
+
+  <!-- Get version information -->
+  <Import Project="$(MSBuildThisFileDirectory)..\..\custom.props" />
+
+  <PropertyGroup>
+    <!-- Optional, defaults to main.  Name of the branch which will be used for calculating branch point. -->
+    <PGOBranch>dev/miniksa/pgo</PGOBranch>

```

Comment:
wait... we want a hard reference to your branch? Won't this cause problems with the git shallow depth checkout change?

Reply:
Nope. You're right but I had to test it here. Since it passed, I'll restore it.

Re: the shallow depth thing... oooh you might be right. If it turns out to have broken it, I'll change the fetch depth for this one run type.